### PR TITLE
Upgrade liquibase to 3.5.x version TRUNK-4986

### PR DESCRIPTION
**Pull request by georgejunior**
## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-4986

## Checklist: I completed these to help reviewers :)

- [ x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -278,7 +278,7 @@
 			<dependency>
 				<groupId>org.liquibase</groupId>
 				<artifactId>liquibase-core</artifactId>
-				<version>2.0.5</version>
+				<version>3.5.0</version>
```
Comment (HerbertYiga) [inline pom.xml]: are you upgrading to the latest liquibase core or you need 3.5.x?

Comment (HerbertYiga) [inline pom.xml]: are you upgrading to the latest liquibase core or you need 3.5.x?

Comment (georgejunior) [reply pom.xml]: > are you upgrading to the latest liquibase core or you need 3.5.x?

that is a huge upgrade that can't just work out of the box so i am trying to go down version by version till i get one that compiles successfully

**Comment by ODORA0** [comment]
You should also try squashing your commits for every push https://wiki.openmrs.org/display/docs/Git-Squashing+commits

**Pull request closed without merging**